### PR TITLE
Loa 228 Add viewModel and fragment for the user creation page

### PR DESCRIPTION
**Pull request by JohannesHausch**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -28,5 +28,10 @@
     <string name="mainPage_profileButton">Profil</string>
     <string name="player_level">3</string>
     <string name="mainPage_compass">Kompass</string>
+    <string name="pick_from_gallery">Aus der Galerie wählen!</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="add_image">Bild hinzufügen!</string>
+    <string name="image_saved">Image saved to %1$s</string>
```
Comment (SebastianMaier03) [inline app/src/main/res/values/strings.xml:34]: Never used string: image_saved

**Review by SebastianMaier03 — CHANGES_REQUESTED** [review]
Remove unused string Then ist alright

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (JohannesHausch) [reply app/src/main/res/values/strings.xml]: I deleted the unused string!

**Review by SebastianMaier03 — APPROVED** [review]
LGTM

**Pull request merged**